polaris-pipeline/Common/Common.csproj

```diff
@@ -43,6 +43,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.CognitiveServices.Vision.ComputerVision" Version="7.0.0" />
     <PackageReference Include="Microsoft.Azure.Core.NewtonsoftJson" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.28.0" />
```

Comment:
This is an older version.  Anything newer brings in a dependency (System.Drawing.Common v7.0) which upsets Aspose dlls